lib/decidim/participatory_documents/admin_engine.rb

```diff
@@ -15,6 +15,9 @@ class AdminEngine < ::Rails::Engine
             get :pdf_viewer
             get :edit_pdf
           end
+          member do
+            post :final_publish
```

Comment:
```suggestion
            post :publish
```
We don't have a "publish", so no need for "final_publish".